src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/EnvelopeResponseMapper.java

```diff
@@ -0,0 +1,213 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.model.mapper;
+
+import com.google.common.collect.ImmutableList;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Envelope;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.NonScannableItem;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Payment;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.ScannableItem;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.out.EnvelopeResponse;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.out.NonScannableItemResponse;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.out.PaymentResponse;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.out.ScannableItemResponse;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class EnvelopeResponseMapper {
```

Comment:
Could we use some standard model mapping library instead of this this class ?

Reply:
I am not sure this simple task is worth a new library with the added complexity. I have thought about it and decided against. If the model becomes bigger it's probably worth reconsidering.

Reply:
You don't need to explicitly import any library for this. There is spring bean utils for this.

``` BeanUtils.copyProperties(dot, entity);```

Reply:
But that copies everything without giving me the option to transform properties if I need to. I am generally against that. I prefer explicit copies as they are more flexible. Unless there is really much time to save, which is not the case here as the model is small

Reply:
the less 'magic' the better imho.
It's easier to see what is exactly happening / spot reasons for errors if it's all explicit.